cmd/experimental/kjobctl/test/e2e/slurm_test.go

```diff
@@ -0,0 +1,315 @@
+				"SLURM_ARRAY_TASK_MAX":    "0",
+				"SLURM_CPUS_PER_GPU":      "",
+				"SLURM_SUBMIT_DIR":        "/slurm/scripts",
+				"SLURM_NPROCS":            "1",
+				"SLURM_CPUS_ON_NODE":      "",
+				"SLURM_ARRAY_TASK_MIN":    "0",
+				"SLURM_JOB_NODELIST":      "profile-slurm-xxxxx-0.profile-slurm-xxxxx",
+				"SLURM_JOB_CPUS_PER_NODE": "",
+				"SLURM_JOB_FIRST_NODE":    "profile-slurm-xxxxx-1.profile-slurm-xxxxx",
+				"SLURM_MEM_PER_NODE":      "",
+				"SLURM_JOB_FIRST_NODE_IP": "",
+			},
+			[]map[string]map[string]string{
+				{
+					"c1": {
+						"SLURM_ARRAY_TASK_ID": "0",
+						"SLURM_JOB_ID":        "1",
+						"SLURM_JOBID":         "1",
+						"SLURM_SUBMIT_HOST":   "profile-slurm-xxxxx-0",
+					},
+				},
+			},
+			false,
+		),
+		ginkgo.Entry(
+			"with --first-node-ip",
+			[]string{"--first-node-ip"}, []string{"--array", "1-5%2", "--nodes", "2", "--ntasks", "2"},
+			int32(3), int32(2),
+			map[string]string{
+				"SLURM_NTASKS_PER_NODE":   "2",
+				"SLURM_ARRAY_JOB_ID":      "1",
+				"SLURM_MEM_PER_CPU":       "",
+				"SLURM_GPUS":              "",
+				"SLURM_NNODES":            "2",
+				"SLURM_MEM_PER_GPU":       "",
+				"SLURM_NTASKS":            "2",
+				"SLURM_ARRAY_TASK_COUNT":  "5",
+				"SLURM_TASKS_PER_NODE":    "2",
+				"SLURM_CPUS_PER_TASK":     "",
+				"SLURM_ARRAY_TASK_MAX":    "5",
+				"SLURM_CPUS_PER_GPU":      "",
+				"SLURM_SUBMIT_DIR":        "/slurm/scripts",
+				"SLURM_NPROCS":            "2",
+				"SLURM_CPUS_ON_NODE":      "",
+				"SLURM_ARRAY_TASK_MIN":    "1",
+				"SLURM_JOB_NODELIST":      "profile-slurm-xxxxx-0.profile-slurm-xxxxx,profile-slurm-xxxxx-1.profile-slurm-xxxxx",
+				"SLURM_JOB_CPUS_PER_NODE": "",
+				"SLURM_JOB_FIRST_NODE":    "profile-slurm-xxxxx-1.profile-slurm-xxxxx",
+				"SLURM_MEM_PER_NODE":      "",
+				"SLURM_JOB_FIRST_NODE_IP": "",
+			},
+			[]map[string]map[string]string{
+				{
+					"c1-0": {
+						"SLURM_ARRAY_TASK_ID": "1",
+						"SLURM_JOB_ID":        "1",
+						"SLURM_JOBID":         "1",
+						"SLURM_SUBMIT_HOST":   "profile-slurm-xxxxx-0",
+					},
+					"c1-1": {
+						"SLURM_ARRAY_TASK_ID": "2",
+						"SLURM_JOB_ID":        "2",
+						"SLURM_JOBID":         "2",
+						"SLURM_SUBMIT_HOST":   "profile-slurm-xxxxx-0",
+					},
+				},
+				{
+					"c1-0": {
+						"SLURM_ARRAY_TASK_ID": "3",
+						"SLURM_JOB_ID":        "3",
+						"SLURM_JOBID":         "3",
+						"SLURM_SUBMIT_HOST":   "profile-slurm-xxxxx-1",
+					},
+					"c1-1": {
+						"SLURM_ARRAY_TASK_ID": "4",
+						"SLURM_JOB_ID":        "4",
+						"SLURM_JOBID":         "4",
+						"SLURM_SUBMIT_HOST":   "profile-slurm-xxxxx-1",
+					},
+				},
+				{
+					"c1-0": {
+						"SLURM_ARRAY_TASK_ID": "5",
+						"SLURM_JOB_ID":        "5",
+						"SLURM_JOBID":         "5",
+						"SLURM_SUBMIT_HOST":   "profile-slurm-xxxxx-2",
+					},
+				},
+			},
+			true,
+		),
+	)
+})
+
+func parseSlurmCreateOutput(output []byte, profileName string) (string, string, string) {
```

Comment:
If you follow the below comment this needs to be changed too
```suggestion
func parseSlurmCreateOutput(output []byte, profileName string) (string, string, string, error) {
```

Reply:
Done